README.md

```diff
@@ -0,0 +1,23 @@
+# Mittwald Cache (APCu & OpCodeCache) widget for the TYPO3 dashboard  
+
+## Purpose of this extension
+This extension allows to draw a doughnut chart widget in the TYPO3 dashboard, about the memory usage of your cache system. 
+It will show how much memory is in use and how much is free.
```

Comment:
Maybe emphasize again that this is about _APCu and OPCache_ memory usage, and NOT memory usage in general?